client/landing/stepper/declarative-flow/registered-flows.ts

```diff
@@ -13,6 +13,7 @@ import {
 	VIDEOPRESS_TV_PURCHASE_FLOW,
 	GOOGLE_TRANSFER,
 	REBLOGGING_FLOW,
+	NEW_SITE_MIGRATION_FLOW,
```

Comment:
Oh wait I think we can remove `NEW_` from the constant, too. I shoulda caught that the first time. 🙃

Reply:
I wasn't sure about that since we already have a `MIGRATION_FLOW`. I wanted to make sure we made the difference between the two obvious.

What do you think?

Reply:
Ugh, yeah, good point. Probably keep it to avoid confusion, at least while we still have both names.